CHANGELOG.md

```diff
@@ -129,6 +129,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Add integration testing for duplicate granule workflows. This includes new
     specs and workflows in the `ingestGranule`, `discoverGranules`,
     `lzardsBackup`, `cnmWorkflow`, and `orca` specs.
+- **CUMULUS-4162**
+  - Made an `includeTimestampHashKey` config variable for the `add-unique-granuleId` and `parse-pdr` task to configure whether the hashKey uses timestamp or not
+  - Added documentation to detail the two different `hashKey` configuration options
```

Comment:
The change log can be more specific, like package/function name and documentation page name.
Added an optional includeTimestampHashKey parameter to the generateUniqueGranuleId function in @cumulus/ingest/granule, with a default value of false.
Added an optional includeTimestampHashKey configuration to the add-unique-granuleId and parse-pdr tasks, also with a default value of false.
Added a documentation page titled "Generate Unique GranuleId" to explain the algorithm for generating unique granuleId.